docs/concepts/snapshot.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Snapshot
+headline: Snapshot
+bodyclass: docs
+layout: docs
+sidenav: doc-side-concepts-nav.html
+type: markdown
+---
+<h2 class="top">Snapshot</h2> 
+
+Snapshot is a state of an Aggregate. A snapshot ”sits” in between events in the history, so that you do not have to replay them all. 
+You read events backwards until you encounter a snapshot, then apply it to the Aggregate, and then play trailing events.
```

Comment:
It's not "you" (a developer) who reads the events of an Aggregate. It's an `AggregateStorage` which knows this “trick” of storing aggregates that does all this job. It's transparent to the developer. He does not need to do anything about it.

Reply:
I changed this concept description the following way: 
Snapshot is a state of an Aggregate. A snapshot ”sits” in between events in the history. 
`AggregateStorage` reads events backwards until encounters a snapshot, applies it to the Aggregate, and plays trailing events.